collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyModule.java

```diff
@@ -382,7 +383,12 @@ public AudioRecorder providesAudioRecorder(Application application) {
     }
 
     @Provides
-    public FormSaveViewModel.FactoryFactory providesFormSaveViewModelFactoryFactory(Analytics analytics, Scheduler scheduler, AudioRecorder audioRecorder, CurrentProjectProvider currentProjectProvider, MediaUtils mediaUtils, FormSessionRepository formSessionRepository) {
+    public EntitiesRepositoryProvider provideEntitiesRepositoryProvider(Application application) {
```

Comment:
Doesn't need to be as the provider stores the repositories in `AppState`. I guess I've been trying to avoid using Dagger as much as possible for functionality, but I'm guessing you're just meaning that I could save on object creation with `@Singleton`?